File: src/main/java/com/netcracker/model/Announcement.java
package com.netcracker.model;


import lombok.Data;

import java.util.Date;
import java.util.List;

@Data
public class Announcement {
    private long announcementId;
    private String title;
    private String body;
    private boolean isOpened;
    private Date createdAt;
    private List<Comment> comments;
    private HouseVoting housevoting;
}

File: src/main/angular/app/models/announcement.ts
import {Comment} from "./comment";
import {HouseVoting} from "./house-voting";

export class Announcement {
    announcementId?: number;
    title?: String;
    body?: String;
    isOpened?: Boolean;
    createdAt?: Date;
    comments?: Comment[];
    houseVoting?: HouseVoting;
}
